### PR TITLE
Fb 322 breadcrumbs for ghbs

### PR DESCRIPTION
**Pull request by tmckernan**
issues
https://dfedigital.atlassian.net/browse/FB-322

Update:

- Add breadcrumbs to top of page
- Remove back link

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -24,6 +25,7 @@ group :development, :test do
   gem "debug", platforms: %i[mri windows], require: "debug/prelude"
   gem "i18n-tasks"
   gem "l10nizer", require: false
+  gem "pry"
```
Comment (threedaymonk) [inline Gemfile:28]: Please remove pry from this PR.

**Review by threedaymonk — CHANGES_REQUESTED** [review]

**Review by threedaymonk — COMMENTED** [review]
I have a few suggestions, but I like the approach and the use of `breadcrumbs_on_rails` with a custom renderer.

I would like to see some kind of test that covers the breadcrumbs. It needn't and shouldn't test the entire HTML, but it would be good to test some happy paths and the edge cases around nil.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -2,10 +2,13 @@
   <%= render "shared/dfe_header" %>
   <div class="page-header">
     <%= render "shared/dfe_beta_banner" %>
+
     <div class="main-container dfe-width-container">
+      <nav class="govuk-breadcrumbs" aria-label="Breadcrumb">
+        <%= render_breadcrumbs(builder: govuk_breadcrumb_builder) %>
+      </nav>
```
Comment (threedaymonk) [inline app/views/layouts/application.html.erb:9]: one more!
